ui/src/app/pages/access/module.tsx

```diff
@@ -55,7 +56,14 @@ export const Module = (props: {
   spinnerProps: WithSpinnerOverlayProps;
   status: AccessModuleStatus;
   style?;
-}) => {
+}
+
+export const Module = (props: ModuleProps) => {
+  const [trainingEnabled, setTrainingEnabled] = React.useState(undefined);
+  useEffect(() => {
+    profileApi().trainingsEnabled().then(setTrainingEnabled);
+  }, []);
```

Comment:
This approach is simple but non-performant, as the `trainings-enabled` endpoint is being called many times on this page. Because this code is temporary (~1 month), the page is low traffic, and the endpoint is fast, I think this tradeoff is OK. I'm open to improving it (for example, by lifting state), so let me know if this is a concern.

Reply:
It  would be nice to have comments something like this is temp and once moodle transition is done we will remove this

Reply:
Or maybe re can rename the api to tempTrainingsEnabled?

Reply:
Good ideas. I added comments and changed the API description to indicate this logic is temporary.